internal/cli/get.go

```diff
@@ -120,17 +120,23 @@ func (o *GetOptions) Validate(args []string) error {
 	if len(name) > 0 && len(o.FieldSelector) > 0 {
 		return fmt.Errorf("cannot specify field selector when fetching a single resource")
 	}
-	if o.Summary || o.SummaryOnly {
+	if o.Summary {
+		if kind != DeviceKind && kind != FleetKind {
+			return fmt.Errorf("summary can only be specified when fetching devices or fleets")
+		}
+		if kind == DeviceKind && len(name) == 0 {
```

Comment:
Why is there a need to check the name length ?
If the name length is 0, doesn't it mean we are not fetching a single device ?

Reply:
Right, I had the condition backwards.  Fixed.